### PR TITLE
Mention the integration guide between Chef Server and Automate

### PR DESCRIPTION
**Pull request by alexpop**
The link in this PR requires this to be merged:
https://github.com/chef/chef-web-docs/pull/254

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,8 +170,11 @@ default["audit"] = {
 
 #### Reporting to Chef Visibility via Chef Server
 
-If you want the audit cookbook to retrieve compliance profiles and report to Chef Automate(Visibility) through Chef Server, set the `collector` and `profiles` attributes. For example:
+If you want the audit cookbook to retrieve compliance profiles and report to Chef Automate(Visibility) through Chef Server, set the `collector` and `profiles` attributes.
 
+This requires recent versions of Chef Server and Chef Automate, as well as integration between the two. More details [here](https://docs.chef.io/integrate_compliance_chef_automate.html#collector-chef-server-visibility).
```
Comment (jeremymv2) [inline README.md]: The word "recent" will lose context over time.  Can we just state the versions?

**Comment by alexpop** [comment]
Makes sense, updated to specify versions.
Thanks Jeremy.

**Comment by jeremymv2** [comment]
👍

**Comment by vjeffrey** [comment]
:+1:

**Pull request merged**